app/components/Settings/Scheduler.react.js

```diff
@@ -0,0 +1,262 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import ReactDataGrid from 'react-data-grid';
+import ms from 'ms';
```

Comment:
add `ms` to `devDependencies`

Falcon uses `webpack` to bundle most of if its dependencies and for this reason most of them are listed as `devDependencies`.

Only dependencies that `webpack` is unable to bundle (e.g. node native modules) should be added as `dependencies`.